src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -75,6 +75,9 @@ internal abstract class TexlFunction : IFunction
         // Return true if the function expects lambda arguments, false otherwise.
         public virtual bool HasLambdas => !_maskLambdas.IsZero;
 
+        // Returns true if the function expect identifiers, false otherwise.
+        public virtual bool HasIdentifiers => false;
```

Comment:
>HasIdentifiers [](http://example.com/codeflow?start=28&length=14)

Why do we need this?  (ie, why isn't IsIdentifierParam sufficient)

Reply:
I don't think so. We need functions that have identifiers to be treated like those having lambdas, with specific scopes for these parameters